libs/shared/utils/src/getPaths/getPaths.ts

```diff
@@ -5,7 +5,9 @@ export const getPaths = <LinkEntry extends { isFolder: boolean; slug: string }>(
   items: LinkEntry[],
 ): TGetPaths[] =>
   items
-    .filter(({ isFolder, slug }) => slug && !isFolder && isSlugRestricted(slug))
+    .filter(
+      ({ isFolder, slug }) => slug && !isFolder && !isSlugRestricted(slug),
```

Comment:
Here's how you fix the other issue:

```suggestion
      ({ isFolder, slug }) => 
        slug && !isFolder && !isSlugForPost(slug) && !isSlugRestricted(slug),
```